### PR TITLE
The 'alert' and 'confirm' of plugins support modifying the footer button label

### PR DESCRIPTION
**Pull request by myth929**
1. support 'okLabel' in $alert;
2. support 'okLabel' and 'cancelLabel' in $confirm;
3. support 'okLabel' and 'cancelLabel' in $prompt;
4. fix inputting of $prompt.

### CODE REVIEW

**Comment by Justineo** [comment]
👍

是否可以同时更新 `managers/prompt` 并且更新一下 test case（`packages/veui/test/unit/specs/managers/{alert,confirm,prompt}.spec.js`）？

**Comment by myth929** [comment]
> 👍
> 
> 是否可以同时更新 `managers/prompt` 并且更新一下 test case（`packages/veui/test/unit/specs/managers/{alert,confirm,prompt}.spec.js`）？

好的

**Review by Justineo — CHANGES_REQUESTED** [review]
Support for `prompt` and test cases required

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@ node_modules
 /components
 icon-names.json
 packages/veui-theme-dls-icons/icons/
+.history/
```
Comment (Justineo) [inline .gitignore:12]:
```suggestion
```

**Pull request merged**